File: src/main/java/codereview/simpleorder/SimpleOrderApplication.java
package codereview.simpleorder;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;

@SpringBootApplication
@EnableFeignClients
public class SimpleOrderApplication {

	public static void main(String[] args) {

		SpringApplication.run(SimpleOrderApplication.class, args);
	}
}

File: src/main/kotlin/msa/with/ddd/SimpleOrderApplication.kt
package msa.with.ddd

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@SpringBootApplication
@EnableFeignClients
class SimpleOrderApplication

fun main(args: Array<String>) {

    SpringApplication.run(SimpleOrderApplication::class.java, *args)
}